tests/autoblocks/test_prompt_manager.py

```diff
@@ -36,11 +41,11 @@ def my_template(
 
 class MyExecutionContext(
     PromptExecutionContext[
-        None,
```

Comment:
couldn't figure out how to do optional generics so just switched back to an empty pydantic class